src/components/CountdownTimer/index.tsx

```diff
@@ -0,0 +1,51 @@
+"use client";
+
+import { useCountdownTimer } from "./useCountdownTimer";
+
+export default function CountdownTimer() {
+  const { timeLeft } = useCountdownTimer();
+
+  return (
+    <div className="w-full max-w-full md:max-w-2xl">
+      <div className="backdrop-blur-sm bg-white/85 rounded-xl py-4 md:py-8 shadow-lg">
+        <div className="flex gap-1 md:gap-4 items-end justify-center text-blue-600">
+          <div className="text-center min-w-[3rem]">
+            <div className="text-xs md:text-sm font-bold mb-1 md:mb-3">
```

Comment:
カウントダウン情報を表示しているだけなのであまり影響はないと思いますが、よりセマンティックな構造にするのであればspan要素あたりを使うと良いかなと思いました。
```suggestion
            <span className="block text-xs md:text-sm font-bold mb-1 md:mb-3">
```

Reply:
https://github.com/tskaigi/tskaigi2025-web-site/pull/9/commits/e5013c7c16e7a4e03081de784c001e839a5014b7

ありがとうございます！：も合わせてspanに修正しました